Source/VCGeneration/Split.cs

```diff
@@ -956,7 +959,7 @@ bool ContainsAssert(Block b)
                || c is AssertCmd && splitOnEveryAssert;
       }
       
-      public static List<Split /*!*/> FindManualSplits(Split initialSplit, bool splitOnEveryAssert)
+      public static List<Split /*!*/> FindManualSplits(Split initialSplit, ImplementationRun run, bool splitOnEveryAssert)
```

Comment:
Isn't the run already contained in the `initialSplit` argument? Why pass it as a separate parameter?